src/utils/chartUtils.js

```diff
@@ -0,0 +1,69 @@
+import * as DateUtils from './dateUtils';
+import * as ApiCalls from './apiCalls';
+
+/**
+ *
+ * @param {Object} startDate
```

Comment:
Should `Object` be `Date`?

Reply:
hmm... i thought jsdoc only accepted basic types (string/object/bool/etc) but I can't find anything that says it's bad so sure